packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -85,6 +85,7 @@ export interface WithCheckoutShippingProps {
     updateCheckout(payload: CheckoutRequestBody): Promise<CheckoutSelectors>;
     updateShippingAddress(address: Partial<Address>): Promise<CheckoutSelectors>;
     shouldRenderStripeForm: boolean;
+    moveCheckUpdatedAddressValueWithinDebouncedExperiment: boolean;
```

Comment:
Thought it might be clearer to use a more action-oriented name instead of just an experiment name. For example, we could use `isDebouncedCheckAndUpdateAddress`.